tests/tt_eager/python_api_testing/unit_testing/misc/test_optimized_conv_v2.py

```diff
@@ -216,3 +216,127 @@ def test_optimized_conv_v2(
     logger.debug(f"Passing={passing_pcc}")
     logger.debug(f"Info={info}")
     assert passing_pcc
+
+
+def test_simple(
```

Comment:
I know we're supposed to deprecate this file, I just find this simple test that I can explicitly tweak each parameter to be very useful, if it's ok with you @tt-nshanker